Add CHANGELOG for v0.9.0 (#196)
turbot/steampipe-plugin-github: CHANGELOG.md

## v0.20.0 [2022-09-09]

_Enhancements_

> - Added `visibility` column as the optional key qual filter in `github_my_repository` table. ([#121](https://github.com/turbot/steampipe-plugin-github/pull/121))

Reviewer: ```suggestion
- Added `visibility` column as an optional list key column in `github_my_repository` table. ([#121](https://github.com/turbot/steampipe-plugin-github/pull/121))
```

_Dependencies_

> - Recompiled plugin with [steampipe-plugin-sdk v4.1.6](https://github.com/turbot/steampipe-plugin-sdk/blob/main/CHANGELOG.md#v416-2022-09-02) which includes several caching and memory management improvements. ([#195](https://github.com/turbot/steampipe-plugin-github/pull/195))

Reviewer: Since 4.1.7 is out, should we just move to that one?
